testing/winetricks/APKBUILD

```diff
@@ -1,34 +1,28 @@
 # Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
 # Maintainer:
 pkgname=winetricks
-pkgver=20170101
+pkgver=20190310
 pkgrel=0
 pkgdesc="Winetricks is an easy way to work around problems in Wine"
-url="http://winetricks.org"
-arch="all"
-license="GPL"
-depends=
-depends_dev=
-makedepends="$depends_dev"
-install=
-subpackages="$pkgname-doc"
+url="https://winetricks.org"
+arch="noarch"
+license="LGPL-2.1-or-later"
+options="!check" # Missing depends bashate & shellcheck
+depends="cabextract unzip xmessage" # Not adding wine since there are two pkgs: wine & wine-development
```

Comment:
If wine-development includes `provides="wine"`, this should not be an issue.

Reply:
Thanks had forgotten the provides option. Is rectified now.